cwltool/main.py

```diff
@@ -695,6 +697,18 @@ def main(argsl=None,  # type: List[str]
                 argsl = sys.argv[1:]
             args = arg_parser().parse_args(argsl)
 
+        # If On windows platform, A default Docker Container is Used if not explicitely provided by user
+        if onWindows() and not args.default_container:
+            args.default_container = "ubuntu"
```

Comment:
wrong means different output. It appears `Alpine` and `Busybox` `rev` tool can't handle input larger than certain character and some error occur in their buffer causing output to print multiple times. I checked it manually in a `alpine` docker. We had same behaviour.